### ISSUE TITLE
What irritates you most of all in Open Source?

### ISSUE THREAD

**Issue by robbbz** [sindresorhus/ama#48]
I saw you tweeting about trolling to you and someone else, stupid bug reports, @issuesfromhell, ...

but what irritates you the most?

**Comment by sindresorhus** [comment]
tl;dr Nothing really.

I used to get annoyed a lot by various things in Open Source; trolls. low-quality issues and pull requests, rude users, etc. But you won't last long in Open Source if you let that stuff affect you. Communication on the internet is hard. You have different personalities, cultures, language proficiencies and coding competencies, all trying to communicate together with text. Lots of opportunities for miscommunication and assumptions, see [Hanlon's razor](https://en.wikipedia.org/wiki/Hanlon's_razor). I've found the best way to handle this is with absolute kindness, no matter what. Trolls will hate it, as they're only seeking a reaction. Rude people will respect it. Nice people will appreciate it. And I feel better. Win win.

**Comment by joeyespo** [comment]
This AMA needs an upvote button :+1:
